subject_files_status_for_dpdash.py

```diff
@@ -94,13 +110,43 @@ def phoenix_files_status(phoenix_dir, out_dir):
             'weekday': ''
         })
         
-        subject_df_tmp = pd.concat([subject_df_tmp, pd.DataFrame(subject_series_tmp).T], axis=1)
-        
+        subject_df_tmp = pd.concat(
+                [subject_df_tmp, pd.DataFrame(subject_series_tmp).T],
+                axis=1)
```

Comment:
I am not a big fan of wrapping lines at 80 characters. What does PEP tell us? Couldn't 114 be indented left by one tab to accommodate 115?

Reply:
[4c6ba88](https://github.com/AMP-SCZ/utility/pull/7/commits/4c6ba88a614861cb1b420767868f0a5b26b4be26)